testimony/parser.py

```diff
@@ -51,15 +48,36 @@ def parse(self, docstring=None):
         """
         if docstring is None:
             return {}, {}
+        tokens_dict = {}
         valid_tokens = {}
         invalid_tokens = {}
-        for match in self.token_regex.finditer(docstring):
-            token = match.group(1).strip().lower()
-            value = match.group(2).strip()
+
+        docstring_xml = publish_string(docstring, writer_name='xml')
```

Comment:
Maybe for future reviews, comment what `publish_string` does? It would be easier then to understand what's being returned and used down below.